### PR TITLE
Home screen buttons

### PR DESCRIPTION
**Pull request by Gyssim**
created new view according to the mockups. Different sizes of the labels.

### CODE REVIEW

File: src/ch/imedias/rsccfx/view/RsccHomeView.java [hunk 1]
```diff
@@ -33,6 +38,17 @@
   ImageView requestImgView;
   ImageView supportImgView;
 
+  final HBox requestBox = new HBox();
+  final VBox requestBoxLabels = new VBox();
+  final HBox supportBox = new HBox();
+  final VBox supportBoxLabels = new VBox();
+
+  Label requestTextBig;
```
Comment (martinfrancois) [inline src/ch/imedias/rsccfx/view/RsccHomeView.java]: please change the variable names according to "Field name differentiation" here https://www.cs.technik.fhnw.ch/confluence16/display/VTDESGB/SAD
for example "requestBigLbl"

File: src/ch/imedias/rsccfx/view/RsccHomeView.java [hunk 2]
```diff
@@ -45,31 +61,49 @@ public RsccHomeView(Rscc model) {
...
-    supportViewBtn.textProperty().setValue("I want to assist someone\nSomebody needs my help");
+    requestTextBig = new Label("I need help");
+    requestTextSmall = new Label("Get somebody to help you remotely");
+    supportTextBig = new Label("I want to assist someone");
+    supportTextSmall = new Label("Somebody needs my help");
   }
 
   private void layoutForm() {
+
+    requestTextBig.setFont(new Font(35));
```
Comment (martinfrancois) [inline src/ch/imedias/rsccfx/view/RsccHomeView.java]: please change these in the CSS, else we don't have control over what font is used

**Review by martinfrancois — CHANGES_REQUESTED** [review]
Hey Simon, thanks for the PR! I think it looks great.
Could you please take a look at the SAD I mentioned and change the variable names accordingly? Also it would be great if you could try to move as much as possible to the CSS.

**Comment by l-marchesi** [comment]
Thanks Simon for the new buttons! :+1: 
I'll merge it :)

**Pull request merged**